docs/DeveloperGuide.md

```diff
@@ -838,17 +839,37 @@ The `generate` command works regardless of whether the prompt label is showing `
    2. Enter `help`.<br>
    **Expected output:** A list of commands specific to your role is shown along with their formats.
 
+### Meetings actions
+1. Declaring availability
+    1. Ensure that you are logged in to cOOPer.
+    2. Enter `available [date in dd-MM-yyyy] [time in HH:mm]`.
```

Comment:
Rmb to add the line break thing! (i cant add it here cause it wont show) Its the character which looks like < br > without the spaces. Can refer to line 866